e2e/app-template/template/src/api/config/controllers/config.js

```diff
@@ -6,6 +8,11 @@ module.exports = {
 
     configService.rateLimitEnable(value);
 
+    ctx.send(200);
+  },
+  async resetTransferToken(ctx) {
```

Comment:
Explanation here:

I've added api tokens and transfer tokens to the database reset. But that means we wipe out the transfer token the test runner depends on before each test. This endpoint allows strapi to recreate that token, so we can just hit that url before each database reset to ensure that the thing resetting the database is actually allowed to do so.

Alternative would have been to update the source database but..... eh, I didn't really like that idea. My end goal is to not have useless tooling data in our dataset. In the future, we should find a better method such as using "import" instead of remote transfer, so that a token isn't even required.